src/lib/components/filePicker.svelte

```diff
@@ -111,6 +112,10 @@
             selectFile(file);
         } catch (e) {
             console.error(e);
+            addNotification({
+                type: 'error',
+                message: e.message || 'Failed to upload file'
```

Comment:
there will always be a message from `AppwriteException`. No need for a fallback.